examples/derived_gauges/derived_gauge.go

```diff
@@ -0,0 +1,237 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Command stats implements the stats Quick Start example from:
+//   https://opencensus.io/quickstart/go/metrics/
+// START entire
```

Comment:
Add a line, otherwise START entire will be a part of the godoc.

Reply:
done.